mousestyles/dynamics/__init__.py

```diff
@@ -263,3 +263,74 @@ def get_prob_matrix_small_interval(string_list):
         if sum(Mat_prob[k, :]) != 0:
             Mat_prob[k, :] = Mat_prob[k, :]/sum(Mat_prob[k, :])
     return Mat_prob
+
+
+def mcmc_simulation(mat_list, n_per_int):
```

Comment:
sorry for the non-consistency. However, I want to keep this function name unchanged. A lot of codes need to be modified if I change the function name, and might cause unnecessary bugs. Thanks.